internal/endpoints/payloads.go

```diff
@@ -121,3 +121,37 @@ func RequestIdPayloads(w http.ResponseWriter, r *http.Request) {
 
 	writeResponse(w, http.StatusOK, string(dataJson))
 }
+
+// PayloadArchiveLink returns a response for /payloads/{request_id}/archiveLink
+func PayloadArchiveLink(w http.ResponseWriter, r *http.Request) {
+
+	reqID := chi.URLParam(r, "request_id")
```

Comment:
It might be good to have an `uuid` check here for `request_id` before we pass it down to `storage_broker`. If the check fails, we can send out a bad request to the user. `storage_broker` does have a `uuid` check for `request_id`, but having another one here might save us from making an unnecessary request.

Reply:
+1

Reply:
Added this! :) Good catch